documentation/source/building.rst

```diff
@@ -33,6 +33,10 @@ Make Variables
     ``SIM``
       Selects which simulator Makefile to use.  Attempts to include a simulator specific makefile from :file:`cocotb/share/makefiles/makefile.$(SIM)`
 
+    ``WAVES``
+      Set this to 1 to enable wave traces dump for Aldec and Questa simulator.
+      To get wave traces in Icarus verilog see Simulator Support chapter.
```

Comment:
verilog -> Verilog

Please link to the section you're referring to.

Reply:
To do the link I need an extension (added in the commit)